File: MegoKoraWebApi/Models/Reservation.cs
namespace MegoKoraWebApi.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Reservation")]
    public partial class Reservation
    {
        [Key]
        [Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ReservationID { get; set; }

        [Key]
        [Column(Order = 1)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int ClientID { get; set; }

        [Key]
        [Column(Order = 2)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int SubPlaygroundID { get; set; }

        public int From { get; set; }

        public int To { get; set; }

        public int Year { get; set; }

        public int Month { get; set; }

        public int Day { get; set; }

        public int? NumberOfHours { get; set; }

        [Column(TypeName = "money")]
        public decimal? TotalMoney { get; set; }

        public bool? ReservationISDone { get; set; }

        public virtual Client Client { get; set; }

        public virtual SubPlayground SubPlayground { get; set; }
    }
}

File: Angular/src/app/Models/Reservation.ts
import { Client } from "./client";
import { SubPlayground } from "./SubPlayground";

export class Reservation{
    ReservationID:number;
    ClientID:number;
    SubPlaygroundID:number;
    From:number;
    To:number;
    Year:number;
    Month:number;
    Day:number;
    NumberOfHours:number;
    TotalMoney:number;
    ReservationISDone:boolean;
    Client: Client;
    SubPlayground: SubPlayground; 
}